command/views/show.go

```diff
@@ -0,0 +1,33 @@
+package views
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/command/arguments"
+	"github.com/hashicorp/terraform/plans"
+	"github.com/hashicorp/terraform/states"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+type Show interface {
+	Plan(plan *plans.Plan, baseState *states.State, schemas *terraform.Schemas)
+}
+
+func NewShow(vt arguments.ViewType, view *View) Show {
```

Comment:
this briefly threw me off (because show has a `-json` flag); can you add a comment that `-json` plan rendering is handled separately (or whatever comment is accurate/makes more sense)?

Reply:
Ah, yes, good catch! Added comments to explain this intermediate step and particularly call out the absent JSON code.